src/librc/librc-daemon.c

```diff
@@ -172,7 +173,7 @@ rc_find_pids(const char *exec, const char *const *argv, uid_t uid, pid_t pid)
 				proc_ns[0] = '\0';
 		}
 		free(buffer);
-		if (pid == 0 && strlen(my_ns) && strlen (proc_ns) && strcmp(my_ns, proc_ns))
+		if (pid == 0 && len_my_ns && strlen (proc_ns) && strcmp(my_ns, proc_ns))
```

Comment:
these strlen calls simply want to know if the string is empty.  that can be checked by `my_ns[0]` and `proc_ns[0]` -- no need for strlen at all.